supervisor/misc/tasks.py

```diff
@@ -106,7 +106,7 @@ async def _update_addons(self):
                     addon.latest_version,
                 )
             # Delay auto-updates for a day in case of issues
-            if utcnow() + timedelta(days=1) > addon.latest_version_timestamp:
+            if utcnow() - timedelta(days=1) < addon.latest_version_timestamp:
```

Comment:
This reads more logical to me:

```suggestion
            if utcnow() < addon.latest_version_timestamp + timedelta(days=1):
```